### DISCUSSION TITLE
Meaning of metrics_value in backtesting_forecaster()

### DISCUSSION THREAD

**Discussion by schoulten** [JoaquinAmatRodrigo/skforecast#699] [Q&A] [answered]
The `skforecast` 0.12.1 docs says that `backtesting_forecaster()` return the `metrics_value` as the "*Value(s) of the metric(s).*", but what does it mean if I'm using these settings?

```
from skforecast.ForecasterBaseline import ForecasterEquivalentDate
from skforecast.model_selection import backtesting_forecaster

srw = ForecasterEquivalentDate(offset = 12)

metric, predictions = backtesting_forecaster(
    forecaster = srw,
    y = df_train,
    steps = 12,
    metric = "mean_squared_error",
    initial_train_size = int(df_train.shape[0]*0.75),
    fixed_train_size = False,
    refit = True,
    verbose = True
)
```

Is `metrics_value` the calculated MSE on the last fold of the cross validation? Is it the average MSE across all folds? Or what?

I'll be very glad if someone clarifies this to me please!

**Comment by JoaquinAmatRodrigo** [comment, marked as answer]
Hi @schoulten,
The returned value is the metric calculated over the time period used for backtesting. For example, let's say you have a time series with daily frequency, a forecaster that predicts 7 steps per head, and you want to backtest it over a period of 1 year. The backtesting process will generate the 365 predictions and calculate the metric with them. This is the metric error over the whole year.

**Reply by schoulten** [reply]
Oh, now I get it! Thank you @JoaquinAmatRodrigo and great package!
